Git/commit.sublime-syntax

```diff
@@ -0,0 +1,81 @@
+%YAML 1.2
+---
+# Highlight regular git commits, merge commits, and tags.
+
+name: Git Commit
+file_extensions:
+  - COMMIT_EDITMSG
+  - MERGE_MSG
+  - TAG_EDITMSG
+scope: source.gitcommit
+
+contexts:
+  prototype:
+    - include: comments
+
+  main:
+    - include: commit-message
+
+  branch-line:
+    - match: \bOn branch\s+(.*)
```

Comment:
How to handle localized comments? I just saw git for linux presenting my the comment in german language. So highlighting fails.

Reply:
there aren't that many translation files: <https://github.com/git/git/tree/master/po>

Reply:
`#: wt-status.c:1568`

https://github.com/git/git/blob/6867272d5b5615bd74ec97bf35b4c4a8d9fe3a51/wt-status.c#L1568

Reply:
@FichteFoll: The `_(` in `_("On branch ")`  tells you the text will be translated by gettext before being displayed on the console or printed to COMMIT_EDITMSG.

Reply:
Yes, my first line is the search string to look for in the `.po` files. Unfortunately, github's own search doesn't produce any useful results for these.